Clients/src/pages/movies/SearchPage.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="min-h-screen bg-gray-100">
+    <!-- Header with Logout -->
+    <header class="bg-gradient-to-br from-black to-red-500 shadow">
+      <div class="max-w-7xl mx-auto px-4 py-4 sm:px-6 lg:px-8">
+        <div class="flex justify-between items-center mb-4">
+          <h1 class="text-2xl font-extralight text-gray-300">Movie Search</h1>
+          <Button
+            @click="handleLogout"
+            
+            :loading="movieStore.isLoading"
+            label="logout"
+            variant="primary"
+          />
+        </div>
+        <!-- Search Components -->
+        <div class="flex gap-4 items-end">
+          <Input
+            v-model="searchQuery"
+            @keyup.enter="handleSearch"
+            label="Search for a movie"
+            placeholder="Enter movie title"
+            class="flex-1"
+          />
+          <Button
+            @click="handleSearch"
+            :disabled="!searchQuery.trim() || movieStore.isLoading"
+            :loading="movieStore.isLoading"
+            label="Search"
+            variant="primary"
+          />
+        </div>
```

Comment:
Great job, but why don't we extract this into a SearchComponent instead of commenting it in here?